kotlin-remix-run-router/src/main/generated/remix/run/router/DeferredData.kt

```diff
@@ -10,25 +10,30 @@ package remix.run.router
 
 import js.core.Record
 import web.abort.AbortSignal
+import web.http.ResponseInit
 import kotlin.js.Promise
 
 
 external class DeferredData {
-    var pendingKeys: Any? /* some expression */
+    var pendingKeysSet: Any? /* some expression */
     var controller: Any? /* some expression */
     var abortPromise: Any? /* some expression */
     var unlistenAbortSignal: Any? /* some expression */
-    var subscriber: Any? /* some expression */
+    var subscribers: Any? /* some expression */
     var data: Record<String, Any>
+    var init: ResponseInit?
+    var deferredKeys: Array<String>
 
-    constructor (data: Record<String, Any>)
+    constructor (data: Record<String, Any>, responseInit: ResponseInit = definedExternally)
 
     var trackPromise: Any? /* some expression */
     var onSettle: Any? /* some expression */
-    fun subscribe(fn: (aborted: Boolean) -> Unit): Unit
+    var emit: Any? /* some expression */
+    fun subscribe(fn: (aborted: Boolean, settledKey: String) -> Unit): () -> Boolean
     fun cancel(): Unit
     fun resolveData(signal: AbortSignal): Promise<Boolean>
     /* get done(): boolean; */
```

Comment:
`val`?

Reply:
There is TODO item in README about getters